mst/billing.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Billing on Managed Service for TimescaleDB
+excerpt: Billing and account management in Managed Service for TimescaleDB
+product: mst
+keywords: [billing, invoicing, accounts]
+---
+
+# Billing
+
+By default, all new Managed Service for TimescaleDB services require a credit
+card, which is charged at the end of the month for all charges accrued over that
+month. Each project is charged separately. Your credit card statement records
+the transaction as coming from Aiven, as Aiven provides billing services for
+Managed Service for TimescaleDB.
+
+Managed Service for TimescaleDB uses hourly billing. This charge is
+automatically calculated, based on the services you have been running in your
+project. The price charged for your project includes:
+
+*   Virtual machine
+*   Networking
+*   Backups
+*   Setting up
+
+<highlight type="note">
+Managed Service for TimescaleDB does not charge you for network traffic used by
+your service. However, your application cloud service provider might charge you
+for the network traffic going to or from your service.
+</highlight>
+
+Terminating or powering a service down stops the accumulation of new charges
+immediately. However, the minimum hourly charge unit is one hour. For example,
+if you launch a Managed Service for TimescaleDB service and shut it down after
+40 minutes, you are charged for one full hour.
+
+Migrating to different service plan levels does not incur extra charges. Note,
```

Comment:
```suggestion
Migrating to different service plan levels does not incur extra charges for the migration itself. Note,
```

Maybe this will be clearer? I found this confusing at first because I read the first sentence, did a double take, then read the second and realized "oh, that's what that means"